rest/models/src/main/java/com/connexta/store/rest/models/MetadataInfo.java

```diff
@@ -0,0 +1,95 @@
+package com.connexta.store.rest.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.net.URL;
+import java.util.Objects;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/** Information representing Metadata. */
+public class MetadataInfo {
+
+  @JsonProperty("metadataType")
+  private String metadataType;
```

Comment:
:question: Why do we need both the metadata type and its URI? Shouldn't the URI be enough to locate the Metadata that needs to be moved?

Reply:
Since we have decided Transform will be sending us the modified metacard along with the IRM, we need to know their locations in addition to their types in order to save them properly.

Reply:
:question: Is it because we tie the type to the URL path somewhere? Or is it because there are difference buckets for different types?